x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -1557,14 +1576,21 @@ class AgentPolicyService {
     }
   }
 
-  private packagePoliciesWithoutMultiplePolicies(packagePolicies: PackagePolicy[]) {
+  private packagePoliciesWithoutMultiplePolicies(packagePolicies: PackagePolicy[]): {
```

Comment:
Maybe this name should be changed? it was originally returning only agent policies _without_ multiple policies but now returns also those _with_ multiple policies, so it's confusing.